capa/main.py

```diff
@@ -533,7 +543,14 @@ def get_extractor(
         if os_ == OS_AUTO and not is_supported_os(path):
             raise UnsupportedOSError()
 
-    if format_ == FORMAT_DOTNET:
+    if format_ == FORMAT_CAPE:
+        import capa.features.extractors.cape.extractor
+
+        with open(path, "r", encoding="utf-8") as f:
```

Comment:
`encoding` doesn't have any meaning if the text file is not opened in text mode (`rt`). recommend sticking to binary mode (`rb`) and letting json decode from utf-8, as expected.

```suggestion
        with open(path, "rb") as f:
```